rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -189,7 +189,8 @@ emu_settings::Render_Creator::Render_Creator()
 				}
 			}
 		}
-		std::scoped_lock{mtx}, thread_running = false;
+
+		std::lock_guard{mtx}, thread_running = false;
```

Comment:
Uhm, aren't we supposed to move to scoped_lock with newer c++?

Reply:
dunno, its implementation is more complex, as long as compiled code is the same I can revert it. will test soon.

Reply:
Nevermind its fine.